packages/muelu/src/Interface/MueLu_HierarchyManager.hpp

```diff
@@ -86,20 +86,20 @@ namespace MueLu {
 
   public:
 
-    //!
+    //! Constructor
     HierarchyManager(int numDesiredLevel = MasterList::getDefault<int>("max levels")) :
-        numDesiredLevel_        (numDesiredLevel),
-        maxCoarseSize_          (MasterList::getDefault<int>("coarse: max size")),
-        verbosity_              (Medium),
-        doPRrebalance_          (MasterList::getDefault<bool>("repartition: rebalance P and R")),
-        implicitTranspose_      (MasterList::getDefault<bool>("transpose: use implicit")),
-        fuseProlongationAndUpdate_ (MasterList::getDefault<bool>("fuse prolongation and update")),
-        sizeOfMultiVectors_     (MasterList::getDefault<int>("number of vectors")),
-        // -2 = no output, -1 = all levels
```

Comment:
What was wrong with that comment?

Reply:
It's not wrong, but I have move the documentation to the definition of the class member. Then, it can be picked up by Doxygen.